test/e2e/apimachinery/aggregator.go

```diff
@@ -616,7 +618,7 @@ func TestSampleAPIServer(ctx context.Context, f *framework.Framework, aggrclient
 			framework.Logf("Observed APIService %v with Labels: %v & Condition: %v", wardle.ObjectMeta.Name, wardle.Labels, cond)
 		}
 	}
-	framework.ExpectEqual(foundUpdatedStatusCondition, true, "The updated status condition was not found. %#v", wardle.Status.Conditions)
+	gomega.Expect(foundUpdatedStatusCondition).To(gomega.BeTrue(), "The updated status condition was not found. %#v", wardle.Status.Conditions)
```

Comment:
`if !foundUpdatedStatusCondition framework.Failf("The updated status condition was not found in:\n%s", format.Object(wardle.Status.Conditions, 1))`

will produce a more readable failure message.